omod/src/main/webapp/pages/conditionlist/addCondition.gsp

```diff
@@ -55,7 +55,7 @@
                         formFieldName: "conditionStartDate",
                         label        : "",
                         useTime      : false,
-                        startDate    : new Date(),
+                        endDate    : new Date(),
```

Comment:
@haripriya999 , why have you changed this?

Reply:
@Ruhanga , The issue was to show the past dates instead of future dates,while creating a condition. So changing startDate to endDate fixed the issue.

Reply:
@haripriya999 can you attach screenshots with the effects of your changes to prove that it was fixed?

Reply:
![pic1](https://user-images.githubusercontent.com/36665698/55713665-bab55a00-5a0e-11e9-83e1-77fedde0c970.jpeg)